movabletype/movabletype
# Fix/reuse mysql schema for plugin tests

Refactored MT::Test::Env 
- respect PluginPath passed to ->new()
- store README only when necessary
- allow ```perl -It/lib -MMT::Test::Env -E 'MT::Test::Env->save_schema'``` to update test schema directly
- add a test to see if schema file is up-to-date
- remove common schema/fixture files from plugins/*/t/fixture
